diff --git a/3rdparty/Makefile.am b/3rdparty/Makefile.am
index 662146f..8fb4946 100644
--- a/3rdparty/Makefile.am
+++ b/3rdparty/Makefile.am
@@ -129,7 +129,7 @@ CLEAN_EXTRACTED =	\
 # which cause the packages to get extracted as necessary. We also
 # apply any patches as appropriate.
 %-stamp: %.tar.gz
-	gzip -d -c $^ | tar xf -
+	gzip -d -c $^ | tar xf - --no-same-owner
 	test ! -e $(srcdir)/$*.patch || patch -d $* -p1 <$(srcdir)/$*.patch
 	touch $@
 
diff --git a/3rdparty/libprocess/3rdparty/Makefile.am b/3rdparty/libprocess/3rdparty/Makefile.am
index 9dc5fb5..7365b75 100644
--- a/3rdparty/libprocess/3rdparty/Makefile.am
+++ b/3rdparty/libprocess/3rdparty/Makefile.am
@@ -81,7 +81,7 @@ CLEAN_EXTRACTED =	\
 # which cause the packages to get extracted as necessary. We also
 # apply any patches as appropriate.
 %-stamp: $(BUNDLED_DIR)/%.tar.gz
-	gzip -d -c $^ | tar xf -
+	gzip -d -c $^ | tar xf - --no-same-owner
 	test ! -e $(top_srcdir)/../$*.patch || patch -d $* -p1 <$(top_srcdir)/../$*.patch
 	touch $@
 
